<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.5.0-0-g9aa0fdb  Build Version: 1.5.7004.21443
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="uav300_head">
  <link
    name="world">
  </link>
  <link
    name="uav_link">
    <inertial>
      <origin
        xyz="0.014599427663741 0.0724676211418231 0.550589552982637"
        rpy="0 0 0" />
      <mass
        value="0.399880166519623" />
      <inertia
        ixx="0.00190812327495004"
        ixy="2.49466975116951E-05"
        ixz="2.77670680958346E-07"
        iyy="0.00323868605686356"
        iyz="-3.88185570049305E-06"
        izz="0.00219861258298517" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0.15"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://uav300_head/meshes/uav_base.STL" 
          scale="1.33 1.33 1.33"/>
      </geometry>
      <material
        name="">
        <color
          rgba="0.76 0.75 0.75 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://uav300_head/meshes/uav_base.STL"
          scale="1.33 1.33 1.33" />
      </geometry>
    </collision>
  </link>
  <joint
    name="uav_joint"
    type="fixed">
    <origin
      xyz="0 0 0"
      rpy="0 0 0" />
    <parent
      link="world" />
    <child
      link="uav_link" />
    <axis
      xyz="0 0 0" />
    <safety_controller
      k_velocity="0" />
  </joint>
  <link
    name="head_link">
    <inertial>
      <origin
        xyz="0.00033386993075224 0.0766238028035832 0.00364568473062213"
        rpy="0 0 0" />
      <mass
        value="0.399880166519623" />
      <inertia
        ixx="0.00219861258298517"
        ixy="3.88185570049305E-06"
        ixz="2.77670680958347E-07"
        iyy="0.00323868605686356"
        iyz="-2.49466975116949E-05"
        izz="0.00190812327495004" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0.144"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://uav300_head/meshes/head_Link.STL" 
          scale="1.5 1.5 1.5"/>
      </geometry>
      <material
        name="">
        <color
          rgba="0.352941176470588 0.552941176470588 0.752941176470588 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://uav300_head/meshes/head_Link.STL" 
          scale="1.5 1.5 1.5"/>
      </geometry>
    </collision>
  </link>
  <joint
    name="head_joint"
    type="fixed">
    <origin
      xyz="0 0 0"
      rpy="0 0 0" />
    <parent
      link="uav_link" />
    <child
      link="head_link" />
    <axis
      xyz="0 0 0" />
    <safety_controller
      k_velocity="0" />
  </joint>

  <link
    name="ardrone1_link">
    <inertial>
      <origin
        xyz="0.014599427663741 0.0724676211418231 0.550589552982637"
        rpy="0 0 0" />
      <mass
        value="0.399880166519623" />
      <inertia
        ixx="0.00190812327495004"
        ixy="2.49466975116951E-05"
        ixz="2.77670680958346E-07"
        iyy="0.00323868605686356"
        iyz="-3.88185570049305E-06"
        izz="0.00219861258298517" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0.05"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://uav300_head/meshes/Drone_2.0_Complete_Indoor_02.STL" 
          scale="0.001 0.001 0.001"/> 
      </geometry>
      <material
        name="">
        <color
          rgba="0.752941176470588 0.252941176470588 0.452941176470588 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://uav300_head/meshes/Drone_2.0_Complete_Indoor_02.STL" 
          scale="0.001 0.001 0.001"/>
      </geometry>
    </collision>
  </link>
  <joint
    name="ardrone1_joint"
    type="fixed">
    <origin
      xyz="1000 1000 0"
      rpy="0 0 0" />
    <parent
      link="world" />
    <child
      link="ardrone1_link" />
    <axis
      xyz="0 0 0" />
    <safety_controller
      k_velocity="0" />
  </joint>

  <link
    name="ardrone2_link">
    <inertial>
      <origin
        xyz="0.014599427663741 0.0724676211418231 0.550589552982637"
        rpy="0 0 0" />
      <mass
        value="0.399880166519623" />
      <inertia
        ixx="0.00190812327495004"
        ixy="2.49466975116951E-05"
        ixz="2.77670680958346E-07"
        iyy="0.00323868605686356"
        iyz="-3.88185570049305E-06"
        izz="0.00219861258298517" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0.05"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://uav300_head/meshes/Drone_2.0_Complete_Indoor_02.STL" 
          scale="0.001 0.001 0.001"/> 
      </geometry>
      <material
        name="">
        <color
          rgba="0.752941176470588 0.352941176470588 0.452941176470588 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://uav300_head/meshes/Drone_2.0_Complete_Indoor_02.STL" 
          scale="0.001 0.001 0.001"/>
      </geometry>
    </collision>
  </link>
  <joint
    name="ardrone2_joint"
    type="fixed">
    <origin
      xyz="1000 1000 0"
      rpy="0 0 0" />
    <parent
      link="world" />
    <child
      link="ardrone2_link" />
    <axis
      xyz="0 0 0" />
    <safety_controller
      k_velocity="0" />
  </joint>

  <link
    name="ardrone3_link">
    <inertial>
      <origin
        xyz="0.014599427663741 0.0724676211418231 0.550589552982637"
        rpy="0 0 0" />
      <mass
        value="0.399880166519623" />
      <inertia
        ixx="0.00190812327495004"
        ixy="2.49466975116951E-05"
        ixz="2.77670680958346E-07"
        iyy="0.00323868605686356"
        iyz="-3.88185570049305E-06"
        izz="0.00219861258298517" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0.05"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://uav300_head/meshes/Drone_2.0_Complete_Indoor_02.STL" 
          scale="0.001 0.001 0.001"/> 
      </geometry>
      <material
        name="">
        <color
          rgba="0.752941176470588 0.352941176470588 0.352941176470588 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://uav300_head/meshes/Drone_2.0_Complete_Indoor_02.STL" 
          scale="0.001 0.001 0.001"/>
      </geometry>
    </collision>
  </link>
  <joint
    name="ardrone3_joint"
    type="fixed">
    <origin
      xyz="1000 1000 0"
      rpy="0 0 0" />
    <parent
      link="world" />
    <child
      link="ardrone3_link" />
    <axis
      xyz="0 0 0" />
    <safety_controller
      k_velocity="0" />
  </joint>

  <link
    name="robot1_link">
    <inertial>
      <origin
        xyz="0.014599427663741 0.0724676211418231 0.550589552982637"
        rpy="0 0 0" />
      <mass
        value="0.399880166519623" />
      <inertia
        ixx="0.00190812327495004"
        ixy="2.49466975116951E-05"
        ixz="2.77670680958346E-07"
        iyy="0.00323868605686356"
        iyz="-3.88185570049305E-06"
        izz="0.00219861258298517" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0.5"
        rpy="1.5704 0 0" />
      <geometry>
        <mesh
          filename="package://uav300_head/meshes/robot.stl" 
          scale="0.001 0.001 0.001"/> 
      </geometry>
      <material
        name="">
        <color
          rgba="0.9 0.9 0.0 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://uav300_head/meshes/robot.stl" 
          scale="0.001 0.001 0.001"/>
      </geometry>
    </collision>
  </link>
  <joint
    name="robot1_joint"
    type="fixed">
    <origin
      xyz="1000 1000 0"
      rpy="0 0 0" />
    <parent
      link="world" />
    <child
      link="robot1_link" />
    <axis
      xyz="0 0 0" />
    <safety_controller
      k_velocity="0" />
  </joint>

   <link
    name="robot2_link">
    <inertial>
      <origin
        xyz="0.014599427663741 0.0724676211418231 0.550589552982637"
        rpy="0 0 0" />
      <mass
        value="0.399880166519623" />
      <inertia
        ixx="0.00190812327495004"
        ixy="2.49466975116951E-05"
        ixz="2.77670680958346E-07"
        iyy="0.00323868605686356"
        iyz="-3.88185570049305E-06"
        izz="0.00219861258298517" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0.5"
        rpy="1.5704 0 0" />
      <geometry>
        <mesh
          filename="package://uav300_head/meshes/robot.stl" 
          scale="0.001 0.001 0.001"/> 
      </geometry>
      <material
        name="">
        <color
          rgba="0.6 0.9 0.1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://uav300_head/meshes/robot.stl" 
          scale="0.001 0.001 0.001"/>
      </geometry>
    </collision>
  </link>
  <joint
    name="robot2_joint"
    type="fixed">
    <origin
      xyz="1000 1000 0"
      rpy="0 0 0" />
    <parent
      link="world" />
    <child
      link="robot2_link" />
    <axis
      xyz="0 0 0" />
    <safety_controller
      k_velocity="0" />
  </joint>

   <link
    name="robot3_link">
    <inertial>
      <origin
        xyz="0.014599427663741 0.0724676211418231 0.550589552982637"
        rpy="0 0 0" />
      <mass
        value="0.399880166519623" />
      <inertia
        ixx="0.00190812327495004"
        ixy="2.49466975116951E-05"
        ixz="2.77670680958346E-07"
        iyy="0.00323868605686356"
        iyz="-3.88185570049305E-06"
        izz="0.00219861258298517" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0.5"
        rpy="1.5704 0 0" />
      <geometry>
        <mesh
          filename="package://uav300_head/meshes/robot.stl" 
          scale="0.001 0.001 0.001"/> 
      </geometry>
      <material
        name="">
        <color
          rgba="0.9 0.9 0.2 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://uav300_head/meshes/robot.stl" 
          scale="0.001 0.001 0.001"/>
      </geometry>
    </collision>
  </link>
  <joint
    name="robot3_joint"
    type="fixed">
    <origin
      xyz="1000 1000 0"
      rpy="0 0 0" />
    <parent
      link="world" />
    <child
      link="robot3_link" />
    <axis
      xyz="0 0 0" />
    <safety_controller
      k_velocity="0" />
  </joint>
</robot>
